composer.json

```diff
@@ -0,0 +1,3 @@
+{
+  "name": "roundingwell/care-ops-frontend"
```

Comment:
should there be anything more in here? Or is the bare minimum fine?

Reply:
the other fields we could put are like author, license, keyword etc.  we don't want version here..  couldn't find anything else useful.